### PR TITLE
CSYS-274: Fix stories structure

### PR DESCRIPTION
**Pull request by CavalcanteLucas**
**Link to task:**
https://labcodes.atlassian.net/browse/CSYS-274

**Staging app:**
https://fix-stories-structure--confetti-storybook.netlify.app/

**How to test:**
Access the staging app and check the structure of stories on the left sidebar. Verify if that's in accordance with expected. Also check that the description of props are show in 'Controls' section of each story.

**Description of your solution:**
This PR fix the stories structure, according to the description in the JIRA card (see the task description for details).
Notes:
- Tag structure partially applied. The files are still in branch to be merged.
- 'Email Input' and 'Password Input' are placed into 'Data Inputs/Text Iput' structure.

### CODE REVIEW

File: stories/Content/Tag/DropdownTag/DropdownTag.stories.js [hunk 1]
```diff
@@ -1,10 +1,10 @@
 import React from "react";
 
-import { ICON_TYPES, TAG_COLORS } from "../../confetti-ds/src/constants";
-import { DropdownTag as Component  } from "../../confetti-ds/src";
+import { ICON_TYPES, TAG_COLORS } from "../../../../confetti-ds/src/constants";
+import { DropdownTag as Component  } from "../../../../confetti-ds/src";
 
 export default {
-  title: "Tag/Dropdown Tag",
+  title: "Content/Tag/Dropdown Tag/Dropdown Tag",
```
Comment (falconedani) [inline stories/Content/Tag/DropdownTag/DropdownTag.stories.js:7]:
```suggestion
  title: "Content/Tag/Dropdown Tag",
```

Dropdown Tag stories from Tag QA should use this title too.

File: stories/Content/Tag/RemovableTag/RemovableTag.stories.js [hunk 2]
```diff
@@ -1,10 +1,10 @@
 import React from "react";
 
-import { ICON_TYPES, TAG_COLORS } from "../../confetti-ds/src/constants";
-import { Button, RemovableTag as Component } from "../../confetti-ds/src";
+import { ICON_TYPES, TAG_COLORS } from "../../../../confetti-ds/src/constants";
+import { Button, RemovableTag as Component } from "../../../../confetti-ds/src";
 
 export default {
-  title: "Tag/Removable Tag",
+  title: "Content/Tag/Removable Tag/Removable Tag",
```
Comment (falconedani) [inline stories/Content/Tag/RemovableTag/RemovableTag.stories.js:7]:
```suggestion
  title: "Content/Tag/Removable Tag",
```

Removable Tag stories from Tag QA should use this title too.

File: stories/Content/Tag/SimpleTag/SimpleTag.stories.js [hunk 3]
```diff
@@ -1,10 +1,10 @@
 import React from "react";
 
-import { ICON_TYPES, TAG_COLORS } from "../../confetti-ds/src/constants";
-import { SimpleTag as Component } from "../../confetti-ds/src";
+import { ICON_TYPES, TAG_COLORS } from "../../../../confetti-ds/src/constants";
+import { SimpleTag as Component } from "../../../../confetti-ds/src";
 
 export default {
-  title: "Tag/Simple Tag",
+  title: "Content/Tag/Simple Tag/Simple Tag",
```
Comment (falconedani) [inline stories/Content/Tag/SimpleTag/SimpleTag.stories.js:7]:
```suggestion
  title: "Content/Tag/Simple Tag",
```

Simple Tag stories from Tag QA should use this title too.

File: stories/Messaging/SystemMessage/AlertWithButton.stories.js [hunk 4]
```diff
@@ -1,10 +1,10 @@
 import React from "react";
 
-import { ICON_TYPES } from "../../confetti-ds/src/constants";
-import { Alert as Component } from "../../confetti-ds/src";
+import { ICON_TYPES } from "../../../confetti-ds/src/constants";
+import { Alert as Component } from "../../../confetti-ds/src";
 
 export default {
-  title: "System Message/Alert With Button",
+  title: "Messaging/System Messages/Alert With Button",
```
Comment (falconedani) [inline stories/Messaging/SystemMessage/AlertWithButton.stories.js:7]:
```suggestion
  title: "Messaging/System Messages/Alert",
```

File: stories/Messaging/SystemMessage/AlertWithoutButton.stories.js [hunk 5]
```diff
@@ -1,10 +1,10 @@
 import React from "react";
 
-import { ICON_TYPES } from "../../confetti-ds/src/constants";
-import { Alert as Component } from "../../confetti-ds/src";
+import { ICON_TYPES } from "../../../confetti-ds/src/constants";
+import { Alert as Component } from "../../../confetti-ds/src";
 
 export default {
-  title: "System Message/Alert Without Button",
+  title: "Messaging/System Messages/Alert Without Button",
```
Comment (falconedani) [inline stories/Messaging/SystemMessage/AlertWithoutButton.stories.js:7]:
```suggestion
  title: "Messaging/System Messages/Alert",
```

File: stories/Messaging/SystemMessage/BannerWithButton.stories.js [hunk 6]
```diff
@@ -1,10 +1,10 @@
 import React from "react";
 
-import { ICON_TYPES } from "../../confetti-ds/src/constants";
-import { Banner as Component } from "../../confetti-ds/src";
+import { ICON_TYPES } from "../../../confetti-ds/src/constants";
+import { Banner as Component } from "../../../confetti-ds/src";
 
 export default {
-  title: "System Message/Banner With Button",
+  title: "Messaging/System Messages/Banner With Button",
```
Comment (falconedani) [inline stories/Messaging/SystemMessage/BannerWithButton.stories.js:7]:
```suggestion
  title: "Messaging/System Messages/Banner",
```

File: stories/Messaging/SystemMessage/BannerWithoutButton.stories.js [hunk 7]
```diff
@@ -1,10 +1,10 @@
 import React from "react";
 
-import { ICON_TYPES } from "../../confetti-ds/src/constants";
-import { Banner as Component } from "../../confetti-ds/src";
+import { ICON_TYPES } from "../../../confetti-ds/src/constants";
+import { Banner as Component } from "../../../confetti-ds/src";
 
 export default {
-  title: "System Message/Banner Without Button",
+  title: "Messaging/System Messages/Banner Without Button",
```
Comment (falconedani) [inline stories/Messaging/SystemMessage/BannerWithoutButton.stories.js:7]:
```suggestion
  title: "Messaging/System Messages/Banner",
```

File: stories/Navigation/Sidebar/NarrowSidebarWithFooterAvatar.stories.js [hunk 8]
```diff
@@ -10,7 +10,7 @@ import { Logotype as LogotypeStory } from "./subcomponents/Logotype.stories";
 import { UserAvatar as UserAvatarStory } from "./subcomponents/UserAvatar.stories";
 
 export default {
-  title: "Sidebar/Narrow Sidebar With Footer Avatar",
+  title: "Navigation/Sidebar/Narrow With Footer Avatar",
```
Comment (falconedani) [inline stories/Navigation/Sidebar/NarrowSidebarWithFooterAvatar.stories.js:13]:
```suggestion
  title: "Navigation/Sidebar/Narrow",
```

**Review by falconedani — CHANGES_REQUESTED** [review]
* Wait for Tags QA merge to apply structure changes in its files
* We should keep the `showRoots` as is, not turn into false. Please remove `.storybook/manager.js`
* Please adjust Tags, System Messages, and Narrow so they can group stories by components, like (PR render X desired): 
PR Render
Desired Result

File: stories/Content/Tag/DropdownTag/DropdownTag.stories.js (hunk 1 above)
Comment (CavalcanteLucas) [reply stories/Content/Tag/DropdownTag/DropdownTag.stories.js]: fixed

File: stories/Content/Tag/RemovableTag/RemovableTag.stories.js (hunk 2 above)
Comment (CavalcanteLucas) [reply stories/Content/Tag/RemovableTag/RemovableTag.stories.js]: fixed

File: stories/Content/Tag/SimpleTag/SimpleTag.stories.js (hunk 3 above)
Comment (CavalcanteLucas) [reply stories/Content/Tag/SimpleTag/SimpleTag.stories.js]: fixed

File: stories/Messaging/SystemMessage/AlertWithButton.stories.js (hunk 4 above)
Comment (CavalcanteLucas) [reply stories/Messaging/SystemMessage/AlertWithButton.stories.js]: fixed

File: stories/Messaging/SystemMessage/AlertWithoutButton.stories.js (hunk 5 above)
Comment (CavalcanteLucas) [reply stories/Messaging/SystemMessage/AlertWithoutButton.stories.js]: fixed

File: stories/Messaging/SystemMessage/BannerWithButton.stories.js (hunk 6 above)
Comment (CavalcanteLucas) [reply stories/Messaging/SystemMessage/BannerWithButton.stories.js]: fixed

File: stories/Messaging/SystemMessage/BannerWithoutButton.stories.js (hunk 7 above)
Comment (CavalcanteLucas) [reply stories/Messaging/SystemMessage/BannerWithoutButton.stories.js]: fixed

File: stories/Navigation/Sidebar/NarrowSidebarWithFooterAvatar.stories.js (hunk 8 above)
Comment (CavalcanteLucas) [reply stories/Navigation/Sidebar/NarrowSidebarWithFooterAvatar.stories.js:13]: fixed

**Review by falconedani — APPROVED** [review]
LGTM!

**Pull request merged**